java/ql/src/semmle/code/java/security/ExternalProcess.qll

```diff
@@ -3,28 +3,27 @@ import semmle.code.java.Member
 import semmle.code.java.JDK
 import semmle.code.java.frameworks.apache.Exec
```

Comment:
I don't think anything from these imports is directly referenced anymore, which means that they're just here to activate the abstract class instances. If that's the case then we should make them private and nest them in a private module to make it explicit that they're only imported for their class activation side effects.
```suggestion

private module Instances {
  private import semmle.code.java.JDK
  private import semmle.code.java.frameworks.apache.Exec
}
```